backend/modules/simulation/router.py

```
@@ -1 +1,169 @@
+        result = await service.start_simulation(current_user.id, request.scenario_id)
+        return result
+    except NotFoundError as e:
+        raise HTTPException(status_code=404, detail=str(e))
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.post("/linear-chat-stream")
+async def linear_chat_stream(
+    request: SimulationChatRequest,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)